src/app/qgsmaptoolcircle2points.cpp

```diff
@@ -32,6 +32,11 @@ void QgsMapToolCircle2Points::cadCanvasReleaseEvent( QgsMapMouseEvent *e )
 {
   QgsPoint point = mapPoint( *e );
 
+  if ( !currentVectorLayer() )
+  {
+    notifyNotVectorLayer();
```

Comment:
Can you clear the rubberband please?
![bug webm](https://user-images.githubusercontent.com/7521540/72747979-64d52b80-3bb6-11ea-857e-9e1826d95abf.gif)